src/Page/HelpSelf.elm

```diff
@@ -31,10 +34,59 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ div [] [ text "Help Self page - grid and singles" ]
+        [ header []
+            [ h1 [ css [ pageHeadingStyle ] ] [ text "[cCc] Help Me With.." ]
+            ]
+        , ul [ css [ gridStyle ] ]
+            [ infoLink "COVID-19" "#"
+            , infoLink "Understanding Economic Abuse" "#"
+            , infoLink "Banking" "#"
+            , infoLink "Debt" "#"
+            , infoLink "Housing" "#"
+            ]
         , ul []
             [ li [] [ a [ href (t GetHelpPageSlug) ] [ text "Get Help" ] ]
             , li [] [ a [ href (t NotAlonePageSlug) ] [ text "Read about others" ] ]
             , li [] [ a [ href (t DefinitionPageSlug) ] [ text "What is Economic Abuse" ] ]
             ]
         ]
+
+
+infoLink : String -> String -> Html msg
+infoLink title link =
+    li
+        [ css
+            [ cardStyle
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero threeColumn ]
```

Comment:
Do these get added in addition to the cardStyles?

I wonder if would be clearer if the         `, flex3 zero zero twoColumn` was excluded from the cardStyles batch and added as an either or?

Happy to go with this though if it works and that is how the media thing is supposed to be constructed.